index.bs

```diff
@@ -440,12 +463,14 @@ interface PictureInPictureWindow : EventTarget {
   readonly attribute long width;
   readonly attribute long height;
 
+  readonly attribute Document? document;
+
   attribute EventHandler onresize;
 };
 </xmp>
 
 A {{PictureInPictureWindow}} instance represents a <a>Picture-in-Picture
-window</a> associated with an {{HTMLVideoElement}}. When instantiated, an
+window</a> associated with a {{HTMLElement}}. When instantiated, an
```

Comment:
Please, ignore my comment in another CL regarding this.

Reply:
Okay